src/utils/Utils.ts

```diff
@@ -1,6 +1,8 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+import moment = require('moment-timezone');
```

Comment:
Its 2.99 MB when un-packed dependency per NPM, could we do this on backend?

Reply:
Removed the dependency.